src/data-pipeline/etl-common/src/main/resources/ts/traffic_source_channel_rule_v0.json

```diff
@@ -0,0 +1,389 @@
+[
+  {
+    "id": 1,
```

Comment:
we need `id` and `order` like this: `id: uuid, order: number`.

Reply:
do not want to give the list an `order` field,  will use norm order of the list.   the id field is not order. we can change it  to 'string' type.